src/components/UriViewer.js

```diff
@@ -94,7 +94,7 @@ const PreviewContent = ({ metadata, metadata: { bucket, name }, googleProject })
         )
       ])],
       [isImage(metadata), () => els.label('Image is too large to preview')],
-      () => els.label(`File can't be previewed.`)
+      () => els.label('File can\'t be previewed.')
```

Comment:
I have mixed feelings about this…

1) I don't think we should have contractions in user-visible strings
2) If we are going to have a contraction, the back-ticks are simpler in my opinion.